cds-snc/report-a-cybercrime
# Self harm words

Fixes #1285 

# Description

scan new reports for self harm words

To test:
```
npm run build
node server.js
```
Go to localhost:3000 and fill out form. use words "agile" and "scrum" somewhere

back on your terminal, should report
```
Self harm words detected: agile,scrum
```

# Checklist:


# Required followup work

Currently using a fake list of words. should replace with the real list (either in repo or in App Service environment variable)
